test/conformance/testing/include/uur/fixtures.h

```diff
@@ -839,11 +842,16 @@ struct urUSMDeviceAllocTestWithParam : urQueueTestWithParam<T> {
 
     void TearDown() override {
         ASSERT_SUCCESS(urUSMFree(this->context, ptr));
+        if (pool) {
+            ASSERT_SUCCESS(urUSMPoolRelease(pool));
+        }
```

Comment:
[NIT] I guess we could check for `use_pool` and clear it to false here in addition to calling `urUSMPoolRelease`. I think we could also assert for a case where `use_pool` is true while `pool` is nullptr as that would mean something gone wrong.

Reply:
stuck an assertion on use_pool in there